frontend/src/queries/Group.ts

```diff
@@ -78,7 +78,7 @@ export function useProjectGroupQuery(
     return useQuery<Group | null, Error>({
         queryKey: computed(() => PROJECT_USER_GROUP_QUERY_KEY(toValue(projectId)!)),
         queryFn: () => getGroupWithProjectId(groups.value!, toValue(projectId)!),
-        enabled: !!toValue(projectId) && groups.value !== undefined,
+        enabled: !!toValue(projectId),
```

Comment:
als iemand mij kan zeggen waarom deze query wel juist functioneert als je niet wacht op `groups.value !== undefined` hoor ik het graag

Reply:
Om dit te doen werken moet `enabled` een getter zijn. Nu wordt de waarde van `groups.value` onmiddellijk en eenmalig berekend. Die berekende waarde zal in het begin dus `undefined` zijn.
```ts
enabled: () => !!toValue(projectId) && groups.value !== undefined,
```
Dit zal wel werken. Dit wordt echter [hier](https://github.com/SELab-2/UGent-5/pull/214/files#diff-bf151f2073467687406ff2558134d90b43de51871c516a8562d2289262de8525) ook gefixt. De query wordt daar volledig geherestructureerd dus valt `enabled` gewoon weg.

Reply:
ah ja natuurlijk, ik zal deze aanpassing dan weg doen